build/cocos2d_libs.xcodeproj/project.pbxproj

```diff
@@ -17490,21 +17490,21 @@
 				HEADER_SEARCH_PATHS = "";
 				IPHONEOS_DEPLOYMENT_TARGET = 9.1;
 				LIBRARY_SEARCH_PATHS = (
-					"\"$(SRCROOT)/../external/png/prebuilt/ios\"",
```

Comment:
Very well. I have one game published for tvOS but I will keep using my own fork of cocos2d.